packages/gatsby-cli/src/reporter/redux/internal-actions.ts

```diff
@@ -286,6 +286,26 @@ export const endActivity = ({
     }
   }
 
+  // Show message if they haven't already enabled the feature and the js build
+  // duration is greater than 20 seconds (much longer than normal.
+  if (
+    activity.id === `webpack-develop` &&
+    !process.env.GATSBY_EXPERIMENT_DEVJS_LAZY &&
+    durationMS > 20
+  ) {
+    console.log(activity, durationS)
+    console.log(actionsToEmit)
+    if (durationS > 3) {
+      console.log(`===================
+Try out this
```

Comment:
Would probably slightly tweak the message, but a bit of a nit pick. Something like:

```js
const message = `
Your local development experience is about to get better, faster, and stronger! 

Specifically, your friendly Gatsby maintainers detected slow Webpack bundling times, which we are planning to improve with future releases.

If you're interested in trialing out one of these future improvements *today* which should make your local development experience faster, go ahead and do the following:

GATSBY_EXPERIMENT_DEVJS_LAZY=true gatsby develop # note, this should be GATSBY_EXPERIMENTAL for consistency with other experiments?

Please do let us know how it goes (good, bad, or otherwise) at https://gatsby.dev/lazy-dev-feedback # I didn't create this yet
`
```

Reply:
Future improvement: CLI command / automated way to do this (consider adding to spec for experiments).

```
gatsby experiment lazy-images enable
```

should automatically edit config / package.json / etc. to do this for you.

Reply:
this looks great! The messaging was very WIP so will jump off the above 👍